### ISSUE TITLE
`outline.figures`  and `outline.figures.index` produce the same result if used inside a section

### ISSUE THREAD

**Issue by DmitrySharabin** [h-tex/eleventy-plugin-outline#5]
But it shouldn't since `outline.figures.index` contains a map of _all_ the figures.

**Comment by LeaVerou** [comment]
Nope, I was wrong. `foo.figures.index` will have the same figures as `foo.figures` since `index` is a flattened version of the `OutlineItems` object at hand, but `Figures` is a flat structure, it doesn't have children of children.

`outline.index` should contain all figures, but it will also contain all headings. :/ 

But the main problem is that figures are not assigned to the correct section....
